exporter/datadogexporter/examples/k8s-agent/service.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-account
+  namespace: default
```

Comment:
Could you name this file serviceaccount.yaml ?

Reply:
Yes! Should I just move it to `roles.yaml`? Is that going to be better? I feel it's perhaps not worth its own file.

Reply:
Done!